example/test/pages/startPage.ts

```diff
@@ -7,31 +7,31 @@ export class StartPage {
   selectedItem = element(by.id('selected'));
   header = element(by.id('header'));
 
-  constructor() {
-    browser.get('/index.html');
+  async goTo() {
+    await browser.get('/index.html');
```

Comment:
Funny when a PR looks so familiar because you made the same changes in your own duplicate PR...